generators/client/templates/angular/package.json.ejs

```diff
@@ -172,7 +176,7 @@ Remove it once the issue is closed
   <%_ } _%>
   "scripts": {
     "prettier:format": "prettier --write \"{,src/**/}*.{md,json,ts,css,scss,yml}\"",
-    "lint": "tslint --project tsconfig.json -e 'node_modules/**'",
+    "lint": "eslint . --ext .js,.ts",
```

Comment:
Do you need to exclude node_modules like previously?

Reply:
yes, that's excluded via an entry in the `.eslintignore`